### PR TITLE
Crates reorganization

### PR DESCRIPTION
**Pull request by AshantiMutinta**
## PR summary
Reorganizing crates and making sure they are compatible with crates.io.

## testing/benchmarking notes

( if any manual testing or benchmarking was/should be done, add notes and/or screenshots here )

## followups

( any new tickets/concerns that were discovered or created during this work but aren't in scope for review here )

## changelog

Please check one of the following, relating to the [CHANGELOG-UNRELEASED.md](https://github.com/holochain/holochain-rust/blob/develop/CHANGELOG-UNRELEASED.md)

- [ ] this is a code change that effects some consumer (e.g. zome developers) of holochain core so it is added to the CHANGELOG-UNRELEASED.md (linked above), with the format `- summary of change [PR#1234](https://github.com/holochain/holochain-rust/pull/1234)`
- [x] this is not a code change, or doesn't effect anyone outside holochain core development

### CODE REVIEW

File: crates/cli/Cargo.toml [hunk 1]
```diff
@@ -6,8 +6,8 @@ authors = ["Holochain Core Dev Team <devcore@holochain.org>"]
 [dependencies]
 holochain_core_types = { path = "../core_types" }
 holochain_core = { path = "../core" }
-holochain_common = { path = "../common" }
-holochain_conductor_api = { path = "../conductor_api" }
+holochain_common = { path = "../../common" }
+holochain_conductor_api = { path = "../conductor_lib" }
```
Comment (lucksus) [inline crates/cli/Cargo.toml]: We should also change the crate name to `holochain_conductor_lib`.

Comment (AshantiMutinta) [reply crates/cli/Cargo.toml]: That is a good idea, I will change that right now

**Comment by AshantiMutinta** [comment]
> There is still a core_types dir in the root with a README, probably from a merge?
> 
> Also, are we not moving `logging` and `common` crates into `crates/`?
> 
> @zippy noticed that the root level Cargo.toml went away, maybe we can have that back?
> 
> Otherwise this looks good!

Oh yes I had it in  my stash instead, restored that,

I have moved the logging but I am not so sure about the common, it seems like it is just a supporting cast like test_utils?

**Comment by AshantiMutinta** [comment]
> 
> 
> Looks like root-level Cargo.toml is missing "crates/core_types"?
> 
> It's such a huge PR that it may be hard to ensure that everything is exactly correct before merging, but as long as tests pass and basic cargo/nix commands work I feel we can iron out any remaining details we find when using it more extensively. If it works seamlessly as is then that's a bonus!

Checked and it seems it was that way in develop as well

**Comment by willemolding** [comment]
Is that also a good time to clear some cruft from the repo? I think the `qt/` dir is no longer needed and I moved the `benchmarks/` to the holochain-persistence repo so we can remove that.

**Comment by maackle** [comment]
We might need to pull out the large amount of fmt applied in this PR if possible, and do it at a more opportune time. In particular, we have https://github.com/holochain/holochain-rust/pull/1701 waiting in the wings, which is another PR that makes a huge number of changes, and it would be a shame to have a messy merge just because of fmt. @AshantiMutinta think you can revert the commit that did the fmt?

**Pull request merged**